formats/financial_release/frontend/examples/financial_release.json

```diff
@@ -7,8 +7,10 @@
   "locale": "en",
   "public_updated_at": "2015-12-01T00:00:00.000Z",
   "details": {
+    "header_legal_disclaimer": "header legal disclaimer example text",
```

Comment:
Might be good to make these examples HTML in order to illustrate the point better.